data_safe_haven/config/config.py

```diff
@@ -43,29 +43,25 @@ def validate(self) -> None:
         try:
             validate_aad_guid(self.admin_group_id)
         except Exception as exc:
-            raise ValueError(
-                f"Invalid value for 'admin_group_id' ({self.admin_group_id}).\n{str(exc)}"
-            )
+            msg = f"Invalid value for 'admin_group_id' ({self.admin_group_id}).\n{exc!s}"
+            raise ValueError(msg) from exc
```

Comment:
Should we make these `DataSafeHavenError`s? Maybe `DataSafeHavenInputError` or `DataSafeHavenParameterError`?

Reply:
I think any of those will work. The correct answer is probably what is most useful for the user.

I would say this is out of scope for this PR though. Can you open an issue?